src/components/Homepage/Homepage.jsx

```diff
@@ -5,11 +5,15 @@ import CategoriesList from "./components/CategoriesList";
 import RestaurantList from "./components/RestaurantList";
 
 export const Homepage = () => {
+  const [searchValue, setSearchValue] = React.useState("");
+  const handleSearchValue = (input) => {
+    setSearchValue(input);
+  };
```

Comment:
You don't need this function. It's doing exactly the same thing with `setSearchValue`. So you can pass `setSearchValue` directly where you pass `handleSearchValue`

Reply:
ok, made change